pom.xml

```diff
@@ -33,6 +33,19 @@
     <tag>HEAD</tag>
   </scm>
 
+  <distributionManagement>
```

Comment:
this is already in the parent right: https://github.com/ExpediaGroup/eg-oss-parent/blob/main/pom.xml#L89 Why do you need this.